packages/cells/src/widget.ts

```diff
@@ -1293,8 +1295,41 @@ export class CodeCell extends Cell<ICodeCellModel> {
     if (this.syncScrolled) {
       this.saveScrolledState();
     }
+    this.updatePromptOverlayIcon();
   }
 
+  /**
+   * Update the Prompt Overlay Icon
+   */
+  updatePromptOverlayIcon(): void {
+    const overlay = DOMUtils.findElement(
+      this.node,
+      'jp-OutputArea-promptOverlay'
+    );
+    if (!overlay) {
+      return;
+    }
+    // If you are changing this, don't forget about svg.
+    const ICON_HEIGHT = 24 + 4 + 4; // 4px for padding
```

Comment:
Last question @brijsiyag, should this be `16` following the icon size change?

> Thanks a lot for the patience

```suggestion
    const ICON_HEIGHT = 16 + 4 + 4; // 4px for padding
```

Reply:
Ooh sorry, yes this need to be 16, thanks for informing. I will update it.